### ISSUE TITLE
Build failing

### ISSUE THREAD

**Issue by jvwilliams23** [aurora-multiphysics/VacuumMesher#16]
Hi, 

I tried to build following the instructions, but got some errors which I have fixed in a fork. 

There are some further errors about missing headers for VacuumMeshing (`Utils`) and igl when building the tests (I just commented out `./tests` in CMakeLists.txt for now).  

Josh

**Comment by jvwilliams23** [comment]
Also now I am getting the following error when running the example:

```
[scarf1324@cn1052 VacuumMesher]$ ./build/bin/vacuum -i Examples/example_mesh.e -o example_vac.e --bound_len 2
[0]PETSC ERROR: ------------------------------------------------------------------------
[0]PETSC ERROR: Caught signal number 11 SEGV: Segmentation Violation, probably memory access out of range
[0]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[0]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[0]PETSC ERROR: configure using --with-debugging=yes, recompile, link, and run
[0]PETSC ERROR: to get more information on the crash.
[0]PETSC ERROR: Run with -malloc_debug to check if memory corruption is causing the crash.
```

**Comment by TheBEllis** [comment]
I believe this should have been fixed by #20.
